app.js

```diff
@@ -13,10 +13,33 @@
     coord = position.coords;
 
     announce("Finding nearby sensors");
+    loadSensorsFromCacheAndShowAQI();
+  }
+ 
+  function loadSensorsFromCacheAndShowAQI() {
+    try {
+      const cachedSensors = window.localStorage.getItem('sensors');
+      sensor_data = JSON.parse(cachedSensors);
+      if (sensor_data.version != 1) {
```

Comment:
```suggestion
      if (sensor_data.version !== 1) {
```

This is a real dumb javascript-ism, but `===` and `!==` are preferred since `==` will actually do some conversions. For example: `"1" == 1` evaluates to `true` while `"1" === 1` will evaluate to false, which is more expected.

I don't think it matters much in this case, but a general pattern to follow in JS

Reply:
pushed a fix. thanks.